src/aat/resources/features/F-044/S-044-Prerequisite_CaseUpdate.json

```diff
@@ -0,0 +1,206 @@
+{
+  "title": "should update an existing case with correct inputs",
```

Comment:
file name should either be 
F-044-Prerequisite_CaseUpdate.json or S-282-Prerequisite_CaseUpdate.json

Reply:
Good catch 👍 I would go with F-044_* as it is common for the whole feature file.

Reply:
updated